openfe/protocols/openmm_afe/equil_solvation_afe_method.py

```diff
@@ -535,6 +588,9 @@ def _create(
         )
         self._validate_alchemical_components(alchem_comps)
 
+        # Validate the lambda schedule
+        self._validate_lambda_schedule(self.settings)
```

Comment:
To improve method reproducibility, I would switch this to passing `self.settings.lambda_settings` instead. That way when we make derived classes we can import the staticmethod across to different places and not face typing errors because we're using different protocol settings.